manifests/pool.pp

```diff
@@ -9,6 +9,8 @@
   $pxeserver   = undef,
   $domain_name = undef,
   $static_routes = undef,
+  $default_lease_time = undef,
```

Comment:
Can we validate these as integers or does dhcpd allow these to be non-integers? If so:
```puppet
if $default_lease_time {
    validate_integer($default_lease_time)
}
```